pycbc/waveform/nr_waveform.py

```diff
@@ -181,10 +179,11 @@ def get_param(value):
             curr_h_real = curr_amp * numpy.cos(curr_phase)
             curr_h_imag = curr_amp * numpy.sin(curr_phase)
             curr_ylm = lal.SpinWeightedSphericalHarmonic(theta, phi, -2, l, m)
-            hp += curr_h_real * curr_ylm.real - curr_h_imag * curr_ylm.imag
+            print curr_ylm
+            hp += curr_h_real * curr_ylm.real + curr_h_imag * curr_ylm.imag
             # FIXME: No idea whether these should be minus or plus, guessing
             #        minus for now. Only affects some polarization phase
-            hc += - curr_h_real * curr_ylm.imag - curr_h_imag * curr_ylm.real 
+            hc += + curr_h_real * curr_ylm.imag - curr_h_imag * curr_ylm.real 
```

Comment:
Hi Patricia,

Making that change alone will produce not sane waveforms. I'm going to send you an email with more details!